mypy/main.py

```diff
@@ -451,6 +451,7 @@ def get_init_file(dir: str) -> Optional[str]:
     'python_version': lambda s: tuple(map(int, s.split('.'))),
     'strict_optional_whitelist': lambda s: s.split(),
     'custom_typing_module': str,
+    'mypy_path': lambda s: [p.strip() for p in s.split(',')],
```

Comment:
Sure. At that point, this function could have a test too. How can I add a small unit test? All I can find is big test suites or use-case scenarios, and that's too much to test a pure function 🙂